src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddLogResult.cs

```diff
@@ -0,0 +1,19 @@
+﻿using SFA.DAS.Approvals.InnerApi.Responses;
+
+namespace SFA.DAS.Approvals.Application.BulkUpload.Commands
+{
+    public class BulkUploadAddLogResult
+    {
+        public BulkUploadAddLogResult() { }
+
+        public int LogId { get; set; }
+
+        public static implicit operator BulkUploadAddLogResult(InnerApi.Responses.BulkUploadAddLogResponse response)
+        {
+            return new BulkUploadAddLogResponse
+            {
+                LogId = response.LogId
+            };
+        }
```

Comment:
IS this code called. It's lacking a test (not I worry too much about that in this case), but I can't see where it is being used

Reply:
Hey @Pauljgraham. Yes, youre right, was superfluous code there. Cleaned up it wasn't used.